src/qt/walletmodel.cpp

```diff
@@ -221,7 +221,7 @@ WalletModel::SendCoinsReturn WalletModel::sendCoins(const QList<SendCoinsRecipie
             CScript scriptPubKey;
             scriptPubKey.SetDestination(CBitcoinAddress(rcp.address.toStdString()).Get());
             vecSend.push_back(make_pair(scriptPubKey, rcp.amount));
-			messages += "<MESSAGE>" + AdvancedCrypt(FromQStringW(rcp.Message)) + "</MESSAGE>";
+            messages += "<MESSAGE>" + FromQStringW(rcp.Message) + "</MESSAGE>";
```

Comment:
will do that'll happen before it puts it into <message> and of course will have to happen on the other end as well. best probably just to cut the invalid chars out of it